### DISCUSSION TITLE
The next Pipeline 2 release will be in the week of April 1

### DISCUSSION THREAD

**Discussion by bertfrees** [daisy/pipeline#758] [Announcements]
For now these are the planned features:

- Expose more settings in graphical user interface.
- Make configuration of braille conversion simpler in graphical user interface.
  - Only show relevant script options
  - Improve script options to specify user stylesheet
  - Improve script options to specify output format
- Continue to improve TTS configuration in graphical user interface.
  - Add script options to specify user lexicons.
  - Global setting for specifying (default/base) lexicon
  - Improve script options to specify user stylesheet
- Start making use of the WebSocket protocol in the web service API
- Critical bugfixes, high-priority feature requests, and easy fixes.

We will keep this list up to date.

The planned Github issues can be seen [here](https://github.com/orgs/daisy/projects/4/views/1), or using the [GitHub CLI](https://cli.github.com/): `gh project --owner daisy item-list 4`.

**Comment by marisademeglio** [comment]
Here is a summary of the relevant changes that are required on the UI side to meet these goals:

"Expose more settings in the graphical user interface"

* https://github.com/daisy/pipeline-ui/issues/199
* https://github.com/daisy/pipeline-ui/issues/200

"Make configuration of braille conversion simpler in graphical user interface."
* "Only show relevant script options"
  * https://github.com/daisy/pipeline-ui/issues/198
* "Improve script options to specify user stylesheet"
  * https://github.com/daisy/pipeline-ui/issues/201
* "Improve script options to specify output format"
  * _Nothing on the UI side for this_

"Continue to improve TTS configuration in graphical user interface."
* "Add script options to specify user lexicons."
  * _Nothing on the UI side for this_
* "Global setting for specifying (default/base) lexicon"
  * https://github.com/daisy/pipeline-ui/issues/202
* "Improve script options to specify user stylesheet"
  *   https://github.com/daisy/pipeline-ui/issues/201

"Start making use of the WebSocket protocol in the web service API"
* _Nothing on the UI side currently (but will be in the future)_

"Critical bugfixes, high-priority feature requests, and easy fixes."
* _Nothing on the UI side_
